js/twitterFetcher.js

```diff
@@ -60,7 +60,7 @@
 
   function strip(data) {
     return data.replace(/<b[^>]*>(.*?)<\/b>/gi, function(a,s){return s;})
-        .replace(/class=".*?"|data-query-source=".*?"|dir=".*?"|rel=".*?"/gi,
+        .replace(/class="(?!(tco-hidden|tco-display|tco-ellipsis))+.*?"|data-query-source=".*?"|dir=".*?"|rel=".*?"/gi,
```

Comment:
Yes, at least that was my intention.

I tried many cases. I give you a few examples :

`<div data-query-source="query" dir="dir" rel="rel">content</div>`
returns `<div>content</div>`

`<div class="test-class" data-query-source="query" dir="dir" rel="rel">content</div>`
returns `<div>content</div>`

`<div class="" data-query-source="query" dir="dir" rel="rel">content</div>`
returns `<div>content</div>`

And when it maches one of the 3 :

`<div class="tco-hidden" data-query-source="query" dir="dir" rel="rel">content</div>`
returns `<div class="tco-hidden">content</div>`

`<div class="tco-display" data-query-source="query" dir="dir" rel="rel">content</div>`
returns `<div class="tco-display">content</div>`

`<div class="tco-ellipsis" data-query-source="query" dir="dir" rel="rel">content</div>`
returns `<div class="tco-ellipsis">content</div>`

Let me know if it's fine or if you want me to amend it.

Thanks